crowbar_framework/lib/crowbar/lock.rb

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2011-2013, Dell
+# Copyright 2013-2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Crowbar
+  class Lock
+    attr_accessor :logger, :path, :name
+    attr_reader   :path, :name, :file
```

Comment:
Style/ExtraSpacing: Unnecessary spacing detected.<br>Style/SingleSpaceBeforeFirstArg: Put one space between the method name and the first argument.